packages/@lwc/compiler/README.md

```diff
@@ -54,7 +54,6 @@ const { code } = transformSync(source, filename, options);
     -   `scopedStyles` (type: `boolean`, optional) - True if the CSS file being compiled is a scoped stylesheet. Passed to `@lwc/style-compiler`.
     -   `enableStaticContentOptimization` (type: `boolean`, optional) - True if the static content optimization should be enabled. Passed to `@lwc/template-compiler`.
     -   `customRendererConfig` (type: `object`, optional) - custom renderer config to pass to `@lwc/template-compiler`. See that package's README for details.
-    -   `enableLwcSpread` (type: `boolean`, default: `false`) - The configuration to pass to the `@lwc/template-compiler`.
```

Comment:
As discussed verbally, let's keep the parameter to avoid breaking people, but make it do nothing. (Similar to `minify`)

Reply:
In addition to this, we should add a warning explaining that this flag will be removed in a future version.

In the future, we might also want to keep all the experimental flags in a separate object to indicate that this flag will be removed once the feature graduates to GA.